src/users/usersReducers.js

```diff
@@ -29,9 +32,15 @@ export default (state: UsersState = initialState, action: Action): UsersState =>
     case ACCOUNT_SWITCH:
       return initialState;
 
-    case INIT_USERS:
-      return action.users.map(mapApiToStateUser);
+    case INIT_USERS: {
+      const users = action.users.map(mapApiToStateUser);
 
+      // TODO: improve and add tests
```

Comment:
Makes sense! 👍 
in my testing I somehow found cases when the app is reloaded and the user data hasn't changed(which I agree is rare); this caused the re-render of MessageContainer and in turn  `renderHtmlChildren` was called again passing in new `children` to MessageFull and MessageBrief. 
After refactoring this into HtmlChildrenContainer, I think we could remove this check, as even in the rare cases; only the container (MessageContainer) will be re-rendered but MessageFull and MessageBrief won't be affected.